lib/ansible/modules/database/influxdb/influxdb_retention_policy.py

```diff
@@ -40,6 +41,11 @@
         description:
             - Determines how many independent copies of each point are stored in the cluster
         required: true
+    shard_duration:
+        description:
+            - Determines the size of a shard group
+        required: false
+        version_added: 2.8
```

Comment:
missing default doc: in arg spec the default is specified as 1d. required: false can be omited (that is the default) and version should be quoted (otherwise it is handled as float)

~~~yaml
    shard_duration:
        description:
            - Determines the size of a shard group
        version_added: "2.8"
        default: 1d
~~~